pre-docs/bolt_configuration_options.md

```diff
@@ -68,6 +68,39 @@ to avoid writing them to disk.
 
 `user`: Login user. Default is `root`.
 
+### OpenSSH configuration options 
+
+In addition to the ssh transport options defined in Bolt-specific configuration files some additional ssh options are read from OpenSSH configuration files ( `~/.ssh/config`, `/etc/ssh_config`, and `/etc/ssh/ssh_config`). Not all OpenSSH configuration values have equivalents in Bolt. Below is a list of options configurable in OpenSSH files.
+ 
+- `User`
+- `Port`
+- `UserKnownHostsFile`
```

Comment:
I thought I left this comment last night, i guess i refreshed... Anyway I started enumerating these and was wondering if it is worth trying to get every single one (this is an incomplete list).